docs/glossary/template.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: landing-page
+sectionKey: Glossary
+eleventyNavigation:
+  parent: Glossary
+title: Template
+theme: Presentation layer
+order: 1
+---
+A template is a pre-designed layout that defines the structure, style and components of a content type.
```

Comment:
Nitpick: If a layout is e.g. single column or two-thirds columns, I'd define a template as a thing that contains layouts and components of a content type. This implies that a template is a specific kind of layout, which I think conflicts with the definition of a layout.